src/service/video-manager-impl.js

```diff
@@ -325,6 +364,44 @@ class VideoEntry {
   }
 
   /**
+   * Called when the orientation of the device changes
+   * @private
+   */
+  orientationChanged_() {
+
+    const vidElement = this.video.element.querySelector('iframe, video');
+
+    // Determine which implementation of the orientation angle to use
+    const angle = this.ampdoc_.win.orientation !== undefined ?
+                  this.ampdoc_.win.orientation : screen.orientation.angle;
+
+    if (this.isVisible_
```

Comment:
reverse and return early (before the angel calculation). Also make sure you bail out if `!viewerForDoc(this.ampdoc_).isVisible()`